### DISCUSSION TITLE
Commenti Disqus

### DISCUSSION THREAD

**Discussion by gbvitrano** [opendatasicilia/opendatasicilia.it#40] [Domande]
Nel vecchio sito i commenti erano gestiti da direttamente da WP o Disqus?

**Comment by dennisangemi** [comment]
Questa mi sembra una domanda per @aborruso o @picomiles

**Comment by aborruso** [comment]
> Nel vecchio sito i commenti erano gestiti da direttamente da WP o Disqus?

Da WP. Sono stati già importati tutti nel nuovo spazio
